cf_units/latex.py

```diff
@@ -0,0 +1,56 @@
+# (C) British Crown Copyright 2019, Met Office
+#
+# This file is part of cf-units.
+#
+# cf-units is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cf-units is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with cf-units.  If not, see <http://www.gnu.org/licenses/>.
+
+try:
+    import cf_units._udunits2_parser.graph as graph
+    from cf_units._udunits2_parser import parse as _parse
+except SyntaxError:
```

Comment:
I'm tempted to suggest an if statement with ``six.PY2`` in it. That is going to be the surest way of us finding this when we make the code py3 only.